### ISSUE TITLE
Java SE 6 required on OS X

### ISSUE THREAD

**Issue by gnilrets** [ondra-m/ruby-spark#28]
I ran `ruby-spark build` successfully, but then when I tried `ruby-spark shell` I get a popup saying I need to install Java SE 6 runtime.

I installed Java 6, and now it's working, but I'm very curious why Java 6 is needed?

**Comment by ondra-m** [comment]
Make sure that `$JAVA_HOME` is correctly set.

**Comment by gnilrets** [comment]
Yep, it's set to `/Library/Java/JavaVirtualMachines/jdk1.8.0_51.jdk/Contents/Home`.  Still asks for Java 6 though.  I didn't change any environment variables after installing Java 6, but that made it start working again.

**Comment by ondra-m** [comment]
You've install Java 6? On README is: "java supported: 7+"

Maybe there is a problem.

**Comment by gnilrets** [comment]
I had Java 8 installed.  But it only worked after I installed Java 6 (without changing JAVA_HOME - JAVA_HOME still points to the Java 8 path).

**Comment by jsmestad** [comment]
Have same issue

**Comment by ondra-m** [comment]
Do you have more informations about this? Try use `--trace`.

I've never had this problem (on debian OS).

**Comment by ondra-m** [comment]
No. Everything is built on selected versions.
